test/api/v3/integration/tasks/POST-tasks_score_id_direction.test.js

```diff
@@ -0,0 +1,121 @@
+import {
```

Comment:
@crookedneighbor most of this aren't already tested in common when we test the score function? Would it be enough to just test that the right params are passed to the score function?